content/guides/models/track/runs/multiple-runs-per-process.md

```diff
@@ -57,7 +56,7 @@ W&B does not support `create_new` mode for integrations that assume a single glo
    wandb.setup(wandb.Settings(reinit="<create_new|finish_previous|return_previous>"))
    ```
 
-- Specify the desired value for `reinit` in an environment variable `WANDB_REINIT`. Defining an environment variable applies the reinit option to `wandb.init()` calls.
+- Specify the desired value for `reinit` in the environment variable `WANDB_REINIT`. Defining an environment variable applies the `reinit` option to `wandb.init()` calls.
```

Comment:
```suggestion
- Specify the desired value for `reinit` in the environment variable `WANDB_REINIT`. Each  `wandb.init()` call reads the environment variable if it is set.
```

Reply:
Good question, I am unsure if this is true....

Reply:
(Leaving out for now)